Parsers/ASimRegistryEvent/test/TrendMicro_VisionOne_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,26 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [ParentProcessId]"
```

Comment:
This is a mandatory field. Please map and also update the Class to Mandatory in tester csv
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-registry-event

Reply:
Done